dev-utils/karma.js

```diff
@@ -56,6 +56,11 @@ const baseConfig = {
     'karma-webpack',
     'karma-sourcemap-loader'
   ],
+  client: {
+    jasmine: {
+      random: false
```

Comment:
jasmine 3.0 has this breaking change of running tests in random order by default. Had to disable it to fix the XHR tests.